developer-docs-site/docs/nodes/validator-node/operator/connect-to-aptos-network.md

```diff
@@ -5,40 +5,11 @@ slug: "connect-to-aptos-network"
 
 # Connecting to Aptos Network
 
-This document describes how to connect your running validator node and public fullnode to an Aptos network. Follow these instructions only if your validator is meeting the minimal staking requirement. (1M APT token)
+This document describes how to connect your running validator node and public fullnode to an Aptos network. Follow these instructions only if your validator has met the minimal staking requirement. 
 
-## Initializing the staking pool
```

Comment:
Why delete this?

Reply:
Moved this to the new staking pool doc. See it in this same PR.